source/ocsp-support/ocsp-support.rst

```diff
@@ -298,10 +308,10 @@ For drivers with finer-grain control over their OCSP behavior, the
 suggested OCSP behavior was chosen as a balance between security and
 availability, erring on availability while minimizing network round
 trips. Therefore, in the `Suggested Stapled OCSP
-Behavior <#suggested-ocsp-behavior>`__ and `Suggested Non-Stapled OCSP
-Behavior <#_win0ckq3thfv>`__ sections, drivers are advised not to reach
-out to OCSP endpoints and CRL distribution points to minimize network
-round trips.
+Behavior <#suggested-ocsp-behavior>`__ section, in order to minimize
```

Comment:
Removed broken link and clarified the wording regarding the round-trip rationale

Reply:
Same comment as above re: single underscore.